tests/Functional/tests/guilabel/guilabel.html

```diff
@@ -1,3 +1,3 @@
 <p>Lorem ipsum dolor sit amet <span class="guilabel">Foo Bar</span>.</p>
-<p>Lorem ipsum dolor sit amet<span class="guilabel">Foo Bar</span>.</p>
+<p>Lorem ipsum dolor sit amet:guilabel:<code>Foo Bar</code>.</p>
```

Comment:
This is a change in behavior, but it's compatible with Sphinx. It also detects ``amet:guilabel:`something` `` as being 2 separate nodes: `amet:guilabel:` and `` `something` ``.